wbooks/src/utils/stringUtils.test.js

```diff
@@ -0,0 +1,14 @@
+import { getInitials, keepNumbers } from './stringUtils';
+
+describe('stringUtils', () => {
```

Comment:
```suggestion
describe('getInitials', () => {
```
Write the function you are testing here